.github/ISSUE_TEMPLATE/--bug-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "🐛Bug report"
+about: Report a problem with Spectrum CSS
```

Comment:
Maybe just "report a bug"

Reply:
It renders like this, so I think `Bug report` with the description works well here: 

![image](https://user-images.githubusercontent.com/201344/65180046-dd618b00-da10-11e9-905e-3cac9124b683.png)

Reply:
ah cool to see the context! this might make it more consistent.